### PR TITLE
add ci test on alpine

### PR DESCRIPTION
**Pull request by stesie**

### CODE REVIEW

File: .github/workflows/build-test.yml [hunk 1]
```diff
@@ -94,3 +94,27 @@ jobs:
                     ./configure --with-v8js=/opt/v8/self-built LDFLAGS="-lstdc++" CPPFLAGS="-DV8_COMPRESS_POINTERS -DV8_ENABLE_SANDBOX"
                     make
                     make test
+
+    alpine:
+        runs-on: ubuntu-latest
```
Comment (JoshuaBehrens) [inline .github/workflows/build-test.yml:108]: Is this how one runs alpine on GitHub Workflows? :D weird concept but if it works

**Comment by JoshuaBehrens** [comment]
Feels like we want to have the phpt test results as artifact on the run so one can view them? One should be able to get into this without having to run it locally again.

> Do you want to save this report in a file? [Yn]: Report saved to: /home/runner/work/v8js/v8js/php_test_results_20240929_1624.txt

I assume something like this would be beneficial to better understand issues reported online
```yaml
      - name: Archive test results
        if: always()
        uses: actions/upload-artifact@v4
        with:
          name: phpt-test-results
          path: php_test_results*.txt
```

**Comment by stesie** [comment]
d'accord, collection makes sense.  I've added it in failure case only though.
Seems to work fine -> https://github.com/phpv8/v8js/actions/runs/11094647649/job/30822343118

And I've compiled 11.3.244.8 on my Ubuntu machine "quickly", and there the test fails as well. Seems like an issue with V8 itself, that the setter is invoked twice there. Unfortunate that it ocurs on exactly the version that's shipped with alpine, but I would just ignore it for the moment (i.e. skip this particular test for that V8 version)

File: .github/workflows/build-test.yml (hunk 1 above)
Comment (stesie) [reply .github/workflows/build-test.yml:108]: yeah, found it weird as well :)

... but there's no runner image with alpine

**Pull request merged**